roofit/roofitcore/src/RooFormula.cxx

```diff
@@ -331,6 +331,21 @@ std::string RooFormula::reconstructFormula(std::string internalRepr) const {
   return internalRepr;
 }
 
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
```suggestion
namespace {

////////////////////////////////////////////////////////////////////////////////
```

Doesn't need to be a private member of `RooFormula`, no? It can be a free function in the anonymous namespace, you just need a `RooArgList const& args` argument for the `_origList`. Like this, we don't clutter the `RooFormula` interface with implementation details.

Reply:
sounds good. The original "reconstructFormula" we keep it though in the header, right? (I had just put it there for parallelism).

Reply:
If it can also be converted to a free function easily, it is better to move it too for consistency. I think static helper functions should be preferred over private member functions, because it makes it immediately clear what data members are used in the call, also also you can be immediately sure that no private data members are used. And also there are no exposed symbols and the user can't access these functions with `#define private public`.